Hazelcast.Net/Hazelcast.IO.Serialization/HeapData.cs

```diff
@@ -87,14 +86,19 @@ public int GetTypeId()
             {
                 return SerializationConstants.ConstantTypeNull;
             }
-            return Bits.ReadIntB(data, TypeOffset);
+            return Bits.ReadIntB(_data, TypeOffset);
         }
 
         public int GetHeapCost()
         {
             // reference (assuming compressed oops)
-            int objectRef = Bits.IntSizeInBytes;
-            return objectRef + (data != null ? ArrayHeaderSizeInBytes + data.Length : 0);
+            var objectRef = Bits.IntSizeInBytes;
+            return objectRef + (_data != null ? ArrayHeaderSizeInBytes + _data.Length : 0);
+        }
+
+        public bool IsPortable()
```

Comment:
good question, this method seems to be ported from the Java Client, and seems only used in some test related code. I'll leave it for now in case it will be useful in the future..